include/point_one/polaris_asio_client.h

```diff
@@ -209,12 +209,25 @@ class PolarisAsioClient {
         VLOG(3) << header;
       }
 
-      // Parse json.
-      boost::property_tree::ptree pt;
+      // Get the entire reply from the socket.
+      std::stringstream reply_body;
```

Comment:
Any particular reason to use a stringstream and not just a string?

Reply:
I use stringstream over strings for concatenations, I think it's also what ptree requires.